src/controller/AutoCommissioner.cpp

```diff
@@ -265,6 +265,10 @@ CHIP_ERROR AutoCommissioner::SetCommissioningParameters(const CommissioningParam
         // The values must be valid now.
         memcpy(mICDSymmetricKey, params.GetICDSymmetricKey().Value().data(), params.GetICDSymmetricKey().Value().size());
         mParams.SetICDSymmetricKey(ByteSpan(mICDSymmetricKey));
+        if (params.GetICDStayActiveDurationMsec().HasValue())
+        {
+            mParams.SetICDStayActiveDurationMsec(params.GetICDStayActiveDurationMsec().Value());
+        }
```

Comment:
This whole block is not needed.  We did `mParams = params;` above, which copied all the state that does not involve buffer pointers.  All this other code is just dealing with copying over buffers, but the stay active duration does not need that.

Reply:
thanks, just find mParams = params. removed.

Reply:
This comment was not actually addressed.